calico_node/startup/startup.go

```diff
@@ -86,18 +86,20 @@ func main() {
 	// updated IP data and use the full list of nodes for validation.
 	node := getNode(client, nodeName)
 
-	// Configure and verify the node IP addresses and subnets.
-	checkConflicts := configureIPsAndSubnets(node)
+	if os.Getenv("CALICO_NETWORKING_BACKEND") == "none" {
```

Comment:
Also, can you add a comment before this line so we know why we're skipping setting IP and checking conflict?